docs/300_OBSERVER.md

```diff
@@ -0,0 +1,76 @@
+# Generic Observer
+An `Observer` evaluates upkeep state from a chain registry and collects upkeeps
+that need to be performed along with data required for comparison and reporting.
+
+Examples can include:
+- polling observer (polls the chain at each block)
+- event trigger observer (logs can trigger observations)
+- scheduled observer (polling occurs on a defined schedule like cron)
```

Comment:
Is it correct to say that observer can be of different types and can be triggered differently but the plugin will always poll observer in each round?

Reply:
Correct. It can be different types or the plugin can have multiple observers. It's all the same. The `Plugin` is responsible to managing rounds, building reports, and feeding accepted reports/transmitted reports back to the observers.